ddtrace/_trace/trace_handlers.py

```diff
@@ -1011,24 +1113,28 @@ def _on_asgi_websocket_receive_message(ctx, scope, message):
     span.set_metric(websocket.MESSAGE_FRAMES, 1)
 
     if hasattr(ctx, "parent") and ctx.parent.span:
+        handshake_span = ctx.parent.span
+        link_attributes = {SPAN_LINK_KIND: SpanLinkKind.EXECUTED}
+
+        _add_websocket_span_pointer_attributes(
+            link_attributes, integration_config, handshake_span, scope, is_incoming=True
+        )
+
         span.set_link(
```

Comment:
nit: we could use link_span here? It could simplify the logic and also set the tracestate on the link (if we want that)